autogpt_platform/backend/backend/blocks/smart_decision_maker.py

```diff
@@ -523,7 +523,6 @@ async def run(
             credentials=credentials,
             llm_model=input_data.model,
             prompt=prompt,
-            json_format=False,
```

Comment:
explicit change to force_json_output=False?

Reply:
`force_json_output` is now default `False` so no need to specify it explicitly